contrib/storage-hive/core/src/test/java/org/apache/drill/exec/fn/hive/TestInbuiltHiveUDFs.java

```diff
@@ -43,4 +47,17 @@ public void testEncode() throws Exception {
         .baselineValues(new Object[] { null })
         .go();
   }
+
+   @Test // DRILL-4459
+   public void testGetJsonObject() throws Exception {
+        setColumnWidths(new int[]{260});
+        String query = "select * from hive.simple_json where GET_JSON_OBJECT(simple_json.json, '$.DocId') = 'DocId2'";
+        List<QueryDataBatch> results = testSqlWithResults(query);
+        String expected = "json\n" + "{\"DocId\":\"DocId2\",\"User\":{\"Id\":122,\"Username\":\"larry122\",\"Name\":" +
```

Comment:
Can you specify this baseline as a complex object instead of a string? The testBuilder can be used to check results against java POJOs and it includes helper methods listOF/mapOf for building up complex structures.